int64 heal
int64 hospitalise
